tests/ruff.toml

```diff
@@ -15,6 +15,7 @@ lint.extend-ignore = [
     "PLR2004", # magic value comparison is common test pattern
     "PLR0913", # too many arguments
     "RUF015", # element index slice is common test pattern
+    "TRY003", # long messages in tests are ok
```

Comment:
Ignoring TRY003 errors (Avoid specifying long messages outside the exception class) in test files